packages/address/src/loader.ts

```diff
@@ -30,7 +29,7 @@ export const loadCountries: (
     | LoadCountriesResponse
     | ResponseError = await response.json();
 
-  if ('errors' in countries) {
+  if (!('data' in countries) && 'errors' in countries) {
```

Comment:
Why remove the `!('data' in countries)` guard?

Reply:
@ismail-syed Because CountryService will return both `data` and `errors` if the requested locale is supported by ShopifyI18n, but is not supported by CountryDb. I will include an example in the description. Thanks for the headsup.  In this case I'll update the LoadCountriesResponse signature to include a optional `errors` field.